spinnaker-monitoring-third-party/third_party/prometheus/install.sh

```diff
@@ -146,6 +155,10 @@ if $DASHBOARDS; then
 fi
 
 if $CLIENT; then
+  # 20170226
+  # Moved this from the daemon requirements for consistency with datadog.
+  pip install prometheus_client
```

Comment:
`sudo pip install` possibly overwrites important files in your python distribution that may be unrecoverable (or difficult to recover from). I think we should use `sudo pip --user spinnaker` or whatever user you want the packages to be installed as. [relevant stackoverflow](http://stackoverflow.com/questions/29310688/sudo-pip-install-vs-pip-install-user).